### PR TITLE
Update metadata on instancehostmap.activate

### PR DESCRIPTION
**Pull request by alena1108**
as its when docker native containers' labels are being populated

https://github.com/rancher/cattle/blob/e9ca0a4f9946483dee4220f9f1f392c330bf7334/code/implementation/docker/compute/src/main/java/io/cattle/platform/docker/process/instancehostmap/DockerPostInstanceHostMapActivate.java#L127

@ibuildthecloud @yasker

### CODE REVIEW

**Comment by yasker** [comment]
Thanks @alena1108 !!

File: code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/MetaDataInfoDaoImpl.java [hunk 1]
```diff
@@ -94,7 +94,8 @@ protected ContainerMetaData map(List<Object> input) {
                 .and(instance.REMOVED.isNull())
                 .and(instance.STATE.notIn(CommonStatesConstants.REMOVING, CommonStatesConstants.REMOVED))
                 .and(exposeMap.REMOVED.isNull())
-                .and(exposeMap.STATE.notIn(CommonStatesConstants.REMOVING, CommonStatesConstants.REMOVED))
+                .and(exposeMap.STATE.isNull().or(
```
Comment (ibuildthecloud) [inline code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/MetaDataInfoDaoImpl.java]: how can state be null?

Comment (alena1108) [inline code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/MetaDataInfoDaoImpl.java]: when exposeMap entry is null. I do left outer join with this table to get service and stand alone containers

Comment (ibuildthecloud) [inline code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/MetaDataInfoDaoImpl.java]: ah hah

**Comment by alena1108** [comment]
@ibuildthecloud all the TFs are caused by the fact that instancehostmap.activate is a part of "agent.instance.services.processes" and "host.agent.instance.services.processes" now (and its the only config item present in both places). TFs happen only with idempotent checks on; network-agent instance start hangs as config items apply gets stuck.

What do you think can be the cause of this problem, and what would be the best way to workaround it?

**Pull request closed without merging**